testApp/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import App from './App.svelte';
+
+const app = new App({
+	target: document.body,
+	props: {
+		bearer: 'Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZ2UiOiIiLCJhcGlrZXkiOiI3NmRiMDRjNi1iMzRkLTQyNjYtYmIwYS03MjMxMjEwZjRlYjkiLCJjb21tdW5pdHlfc3VwcG9ydGVkIjpmYWxzZSwiZXhwIjoxNzI2NDM5NDg4LCJnZW5kZXIiOiIiLCJoaXRzX3Blcl9kYXkiOjEwMDAwMDAwMCwiaGl0c19wZXJfbW9udGgiOjEwMDAwMDAwMCwiaWQiOiIyNDczMjczIiwiaXNfbW9kIjpmYWxzZSwiaXNfc3lzdGVtX2tleSI6ZmFsc2UsImlzX3RydXN0ZWQiOmZhbHNlLCJwaW4iOm51bGwsInJvbGVzIjpbXSwidGVuYW50IjoidHZkYiIsInV1aWQiOiIifQ.P6bF2imk8DtJiGWjYKSuHQsRNquJvB_ciaE6rbAgtpfdLCxtebwNUO8e7_6ADoqg9SppDm8D9GhO9QTbvnBv_ZzI5pWnRcd56akHo5bbfCppYPbeANPu2itTPSNEBbgddb7Gr_WnDSK0ocZDX7gzzxU22llXy05-3jT4C2r6fZqALmcTwjsYfRjrLYv5yEuhLli6nil2ZolOUk-ScfapzLDfhlNzmH8uL4IpZxu666reMRWPvJV_tEpIvIzO4-y0VgfT0UKeIyGnpfuobvxhAqX7oMVa2ZiCt2pjtls9vChsPCuQTp-IszWGWJTkLKMkZ7RCKJqipwACHJ6bzbHzNab2DaR4WQzLQch7Lv8NMfzX-xTaGyJSqoMFoTI74GNadBqc38VfTaelBIi-YQc7TV5LzKbTjpmWJN7qbqKdyh5vpJwcthwrpvB9F3AIBX3PDk6RbTk-asY6qic96a1DSwjnzP2JYOPLHHNJv2kpg3abeUvE7boPbj8MYS_rCaOffAd33jjev0uExW2Q3hHPQNwI3eEsmIua7wl8gKaWSbeJMBCMroQOtnJlx5B9rChdkFtnbUGtaERFTFPaQzxGSXEj3TnePnTI3XHXGnr_L8PZXFLgW1DlXfQ1gHc20qCm8tREMdu2MG-yS0WAzOYzlaoSapxySf1E9C8aGytSTtg'
```

Comment:
An access token should never be committed to GIT. This access token is now compromised and should be regenerated.
An access token should be configured in a configuration file that the developer fills in on their own machine, but never commits. This is normally done with a `.env` file that is ignored by GIT through the `.gitignore` file.

Reply:
I agree, would have been a real problem with real data after migrating from svelte to SvelteKit to make use of the .env file I definitely will not forget this again.

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-9f3c893cdbb3f5eefcdd0c7fc82cca19ddb8ea5033369380bea6472d6649f73e